src/deploy/scripts/signal_run.sh

```diff
@@ -0,0 +1,13 @@
+cd /home/ubuntu/signal/src/api
```

Comment:
- [ ] Add documentation

Reply:
This is a script that can be used to start up the dev server on 8080 if you have to debug anything